pkg/pushsync/pushsync.go

```diff
@@ -209,6 +209,12 @@ func (ps *PushSync) PushChunkToClosest(ctx context.Context, ch swarm.Chunk) (*Re
 	peer, err := ps.peerSuggester.ClosestPeer(ch.Address())
 	if err != nil {
 		if errors.Is(err, topology.ErrWantSelf) {
+			// this is to make sure that the sent number does not diverge from the synced counter
+			t, err := ps.tagg.Get(ch.TagID())
```

Comment:
sigh....not this tagg again...

Reply:
whatss wrongg?

Reply:
we fixed a bunch of these, but it seems this one managed to slip by.

i'll make sure to include this change through #576